no-AA-eviction-RoF.md (#1281)
RedisLabs/redislabs-docs: content/rs/administering/creating-databases/create-active-active.md

    - [**Data eviction policy**]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}}) -
        When the total size of the database reaches its memory limit,
        the database cannot accept new keys. You can select a data eviction policy that defines which keys to delete to make room for new keys. The default eviction policy for Active-Active is `noeviction`.
        
    {{< note >}}

>     

Reviewer: ```suggestion
```

> - Data eviction policy is not supported for Active-Active databases with Redis on Flash (RoF).

Reviewer: ```suggestion
- Data eviction policies are not supported for Active-Active databases with Redis on Flash (RoF).
```

> 

Reviewer: ```suggestion
```
